namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-api-certificates/00-namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "digital-prison-services"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/prisonstaffhub.git"
```

Comment:
guess this should link to auth and also to prison api, either that or leave blank :thinking: